client/src/plugins/camunda-plugin/deployment-tool/DeploymentConfigOverlay.js

```diff
@@ -260,21 +262,18 @@ export default class DeploymentConfigModal extends React.PureComponent {
             return (
               <form onSubmit={ form.handleSubmit }>
 
-                <Modal.Title>
-                  {
-                    title || 'Deploy Diagram to Camunda Platform'
-                  }
-                </Modal.Title>
+                <Overlay.Title>{title || 'Deployment'}</Overlay.Title>
 
-                <Modal.Body>
+                <Overlay.Body>
                   {
                     intro && (
                       <p className="intro">
                         { intro }
                       </p>
                     )
                   }
-                  <fieldset>
+
+                  <fieldset className="fieldset">
```

Comment:
Why is attaching a class / CSS override necessary here?

I'd assume we adopt a uniform way to style forms within overlays.